tests/FakeItEasy.IntegrationTests/FakingClassesTests.cs

```diff
@@ -12,10 +11,7 @@ public class FakingClassesTests
         [Test]
         public void Should_be_able_to_get_a_fake_value_of_uri_type()
         {
-            using (Fake.CreateScope(new NullFakeObjectContainer()))
-            {
-                A.Fake<Uri>();
-            }
+            A.Fake<Uri>();

```

Comment:
> You prefer null-checking the exception to checking that we got a valid Fake back?

We're not really checking that we got a valid Fake back, we're just letting the exception that may be thrown during its creation bubble out to the test framework and therefore implicitly fail the test rather than explicitly failing it using an assertion.